src/IBM.Watson.Discovery.v2/DiscoveryService.cs

```diff
@@ -1825,7 +1828,7 @@ public DetailedResponse<AnalyzedDocument> AnalyzeDocument(string projectId, stri
 
                 if (file != null)
                 {
-                    var fileContent = new StreamContent(file);
+                    var fileContent = new ByteArrayContent(file.ToArray());
```

Comment:
@kevinkowa when we reviewed this one together. I remember that we ignored this change.
I was about to revert back to `var fileContent = new StreamContent(file);` but also found that every other places use `ByteArrayContent`. So I'm wondering if we should just add this change since I did not see any test failure on DiscoveryService. But, I also don't know if it need to be kept as `SteamContent`. Please let me know if we want to convert this part back.

Reply:
PR looks good after this change

Reply:
Okay! I updated the change. Thank you